static/js/protein/chart.ts

```diff
@@ -29,10 +31,63 @@ export function drawTissueScoreChart(
   id: string,
   data: { name: string; value: string }[]
 ): void {
-  // TODO: convert data to tmpData format.
-  const tmpData = [{ x1: 20, x2: 60, y1: 30, y2: 50 }];
+  const testData = data;
```

Comment:
I believe this is an unused line so it can be deleted

Reply:
Done !